internal/telemetry/cluster.go

```diff
@@ -145,6 +145,14 @@ func (c *Collector) IngressClassCount(ctx context.Context) (int, error) {
 	return len(ic.Items), nil
 }
 
+// PolicyCount returns number of Policies watched by NIC.
+func (c *Collector) PolicyCount() int {
```

Comment:
#5453 This could make sense to granularise into counts for the different Policy types

Reply:
Will do in a sepearte PR.